test/jake/jakelib/seed-test.js

```diff
@@ -15,6 +15,75 @@ const KNEX = path.normalize(__dirname + '/../../../bin/cli.js');
 const taskList = [];
 /* * * TESTS * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
+test(
```

Comment:
I would say that for new tests `cli-testlab` tests would be preferable (and easier to write) than using jake directly. See test/cli/migrate-make.spec.js for an example.

Also async/await is preferred over using promise chains.

Reply:
@kibertoad Thanks for the tips. I will use cli-testlab instead. Also, yeah, I definitely didn't want to be using promise chains and Bluebird. I just figured the rest of the codebase was using it, so I should too. So, cool! async await valhalla awaits me